README.md

```diff
@@ -61,7 +59,7 @@ provides the following set of benefits:
 - Dynamic wallpaper which shows Ethernet/WLAN address and hostname
 - Comes with `lirc` preinstalled to run the IR receiver component
 
-You may download the zip-compressed archive `crowpi.img.zip`, extract it and flash it with the imaging tool of your choice to get started.
+Download the zip-compressed archive [main-crowpi.img.zip](https://pi4j-download.com/main-crowpi.img.zip), extract it and flash it with the imaging tool of your choice to get started.
```

Comment:
```suggestion
Download the [latest zip-compressed archive](https://pi4j-download.com/latest.php?flavor=crowpi), extract it and flash it with the imaging tool of your choice to get started.
```

I've added this helper file to the Pi4J download host which will always redirect to the latest version, preferring semver (e.g. `2.3.0` is preferred over `2.2.1`) and falling back to alphabetical name sorting if no versioned releases exist, in our case currently `main`. That way once we have our first actual version up and running (e.g. `1.0.0`), this link will point to this version instead of always pointing towards our `main` branch which might have unexpected changes or sometimes be W.I.P.